plugins/vpp/aclplugin/vppcalls/vpp2001/acl_vppcalls.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	acl "github.com/ligato/vpp-agent/api/models/vpp/acl"
 	"github.com/ligato/vpp-agent/plugins/vpp/aclplugin/vppcalls"
-	aclapi "github.com/ligato/vpp-agent/plugins/vpp/binapi/vpp1901/acl"
+	vpp_acl "github.com/ligato/vpp-agent/plugins/vpp/binapi/vpp2001/acl"
```

Comment:
I'm not sure this is a fitting alias for binapi package, because the [package name for the models is the same](https://github.com/ligato/vpp-agent/blob/master/api/models/vpp/acl/keys.go#L15) even though here it's aliased. It might be confusing to read since both packages are used in vppcalls heavily.

Reply:
I would go for something like `vppapi_acl` or `binapi_acl` which is probably the most clear one.

Reply:
I'd go for `vppapi_acl` (and similar for others)